<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EZTV Torrent Scraper - Find Any Torrent!</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../imgs/torrent_icon.png</normaloff>../imgs/torrent_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>140</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <property name="title">
          <string>Srape For Movie/Show/Subtitle/Socce-Streams/Anime Data</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="keyword_edit">
              <property name="minimumSize">
               <size>
                <width>350</width>
                <height>30</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Enter the keyword you want to search/scrape for.
You can, for example, enter Game Of thrones to get
all the torrents available for GOT and
then later filter the Sraped data for what you want exactly.</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="placeholderText">
               <string>Keyword... Example: Game Of Thrones</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>HINT: Only search ONCE then use the filter option to filter what to download or use. Hover over buttons to get help.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Scrape For : </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="scrape_for_combo">
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Here you the category of media your search is from.</string>
                </property>
                <item>
                 <property name="text">
                  <string>Tv-Shows</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Movies</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Subtitles</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Soccer-Streams</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Anime</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>General</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Server To Scrape From :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="scrape_from_combo">
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Here you select the popular websites from where you
want to scrape the data from.</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="show_status_check">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Keep this option checked to see the scraper working.</string>
              </property>
              <property name="text">
               <string>Show Scrape Status</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="start_scraping_button">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>24</height>
               </size>
              </property>
              <property name="toolTip">
               <string>This button starts the scraping process and will search for
the Keyword in the selected site and return the found data.</string>
              </property>
              <property name="text">
               <string>Start Scraping</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../imgs/scrape.png</normaloff>../imgs/scrape.png</iconset>
              </property>
              <property name="default">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="stop_scraping_button">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="toolTip">
               <string>This button will stop the scraping process.</string>
              </property>
              <property name="text">
               <string>Stop</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../imgs/stop.png</normaloff>../imgs/stop.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Filter The Data You Have Scraped</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>Current Database : </string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="currnet_database_label">
                    <property name="text">
                     <string>Game Of Thones</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="more_information_button">
                  <property name="toolTip">
                   <string>This button will try to get more details of the KEYWORD and display it in your browser.</string>
                  </property>
                  <property name="text">
                   <string>More Information on &lt;Movie Name&gt;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_3">
                <property name="title">
                 <string>Filter Options: </string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_11">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_5">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_9">
                     <item>
                      <widget class="QCheckBox" name="season_check">
                       <property name="toolTip">
                        <string>In case you are scraping for a Tv-Show
this will only filter in the season you set here.</string>
                       </property>
                       <property name="text">
                        <string>Season  </string>
                       </property>
                       <property name="checked">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QSpinBox" name="season_spin">
                       <property name="minimumSize">
                        <size>
                         <width>50</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="toolTip">
                        <string>In case you are scraping for a Tv-Show
this will only filter in the season you set here.</string>
                       </property>
                       <property name="minimum">
                        <number>1</number>
                       </property>
                       <property name="maximum">
                        <number>999</number>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_10">
                     <item>
                      <widget class="QCheckBox" name="episode_check">
                       <property name="toolTip">
                        <string>In case you are scraping for a Tv-Show
this will only filter in the episode you set here.</string>
                       </property>
                       <property name="text">
                        <string>Episode</string>
                       </property>
                       <property name="checked">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QSpinBox" name="episode_spin">
                       <property name="minimumSize">
                        <size>
                         <width>50</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="toolTip">
                        <string>In case you are scraping for a Tv-Show
this will only filter in the episode you set here.</string>
                       </property>
                       <property name="minimum">
                        <number>1</number>
                       </property>
                       <property name="maximum">
                        <number>999</number>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="Line" name="line_2">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_6">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_8">
                     <item>
                      <widget class="QCheckBox" name="search_term_check">
                       <property name="text">
                        <string>Search Term: </string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="search_term_edit"/>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="display_all_data_check">
                     <property name="text">
                      <string>Display All Data</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="Line" name="line_3">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_7">
                   <item>
                    <widget class="QLabel" name="label_5">
                     <property name="text">
                      <string>Sort Results By: </string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="sort_by_combo">
                     <property name="minimumSize">
                      <size>
                       <width>120</width>
                       <height>24</height>
                      </size>
                     </property>
                     <item>
                      <property name="text">
                       <string>Name or Title</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Seeders</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Torrent Size</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="Line" name="line_5">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_5">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>252</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="Line" name="line_6">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="filter_button">
                   <property name="minimumSize">
                    <size>
                     <width>100</width>
                     <height>50</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>60</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="toolTip">
                    <string>Filter the scraped data for what you want according to the set filter options.
This can be done more than once.</string>
                   </property>
                   <property name="text">
                    <string>FIlter</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_4">
                <property name="title">
                 <string>Filter Results : </string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="QTableWidget" name="results_table">
                   <column>
                    <property name="text">
                     <string>Torrent Name</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Torrent Size</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Seeders</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Magnet Link</string>
                    </property>
                   </column>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1032</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>season_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>season_spin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>episode_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>episode_spin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>search_term_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>search_term_edit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
